swagger.yaml

```diff
@@ -1,75 +1,330 @@
+
+        - name: "body"
+          in: "body"
+          description: "Object that will overwrite the card specified by its id"
+          required: true
+          schema:
+            $ref: "#/definitions/NewData"
+      responses:
+        200:
+          description: "Card successfully updated"
+          schema:
+            $ref: "#/definitions/SingleData"
+        400:
+          description: "Invalid syntax in PUT request"
+          schema:
+            $ref: "#/definitions/Error"
+        404:
+          description: "No card with this id"
+          schema:
+            $ref: "#/definitions/Error"
+        500:
+          description: "Internal server error"
+          schema:
+            $ref: "#/definitions/Error"
+
+    delete:
+      tags:
+      - cards
+      summary: "Remove a card specified by its id"
+      operationId: "removeCardById"
+      consumes:
+      - "application/json"
+      produces:
+      - "application/json"
+      parameters:
+        - name: "id"
+          in: "path"
+          description: "Id of card to be removed"
+          required: true
+          type: "integer"
+          format: "int64"
+      responses:
+        200:
+          description: "Card successfully removed"
+          schema:
+            $ref: "#/definitions/SingleData"
+        404:
+          description: "No card with this id"
+          schema:
+            $ref: "#/definitions/Error"
+        500:
+          description: "Internal server error"
+          schema:
+            $ref: "#/definitions/Error"
 
 definitions:
-  Info:
+
+  Card:
+    type: "object"
     properties:
-      name:
-        type: string
-        description: Name of the API.
-      time:
-        type: string
-        description: Build time in ISO 8601 format.
-      unixTime:
-        type: integer
-        description: Build time as milliseconds since the unix epoch.
-      commit:
-        type: string
-        description: Git commit hash of the revision which was built.
-      documentation:
-        type: string
-        description: Filename of the swagger specification for the API.
+      id:
+        type: "integer"
+        format: "int64"
+      type:
+        type: "string"
+      attributes:
+        type: "object"
+        properties:
+          type:
+            type: "string"
+          name:
+            type: "string"
+          color:
+            type: "string"
+          rarity:
+            type: "string"
+          energy:
+            type: "integer"
+          description:
+            type: "string"
+
+  SingleData:
+    type: "object"
+    properties:
+      data:
+        $ref: "#/definitions/Card"
+
+  MultiData:
+    type: "object"
+    properties:
+      data:
+        type: "array"
+        items:
+          $ref: "#/definitions/Card"
+
+  NewData:
+    type: "object"
+    properties:
+      data:
+        type: "object"
+        properties:
+          type:
+            type: "string"
+          attributes:
```

Comment:
The attributes for NewData and Card are the same. Maybe use one shared object instead?